store.go

```diff
@@ -76,8 +76,12 @@ type CookieStore struct {
 //
 // It returns a new session and an error if the session exists but could
 // not be decoded.
-func (s *CookieStore) Get(r *http.Request, name string) (*Session, error) {
-	return GetRegistry(r).Get(s, name)
+func (s *CookieStore) Get(r *http.Request, name string) (*Session, *http.Request, error) {
```

Comment:
We shouldn't need to return the store here, if we're already replacing the `*http.Request` passed in.